README.md

```diff
@@ -145,7 +145,6 @@ docker create \
     --restart always \
     --network host \
     --cap-add NET_ADMIN \
-    --volume /usr/bin/wg:/usr/bin/wg \
```

Comment:
assuming we are now having this one container in control of wireguard aswell? No longer required on the host?